src/coreclr/jit/compiler.h

```diff
@@ -3859,6 +3859,7 @@ class Compiler
     // mechanism passes the address of the return address to a runtime helper
     // where it is used to detect tail-call chains.
     unsigned lvaRetAddrVar;
+    unsigned lvaSwiftSelfArg;
```

Comment:
Since all the places where we check `lvaSwiftSelfArg` are under `#ifdef SWIFT_SUPPORT`, should we ifdef this, and where we initialize it in `Compiler::lvaInit`?

Reply:
Yeah, I'll do that.